### ISSUE TITLE
Node and Link Visibility errors

### ISSUE THREAD

**Issue by Alexithemia** [vasturiano/3d-force-graph#237]
When I set any node or link to be not visible using .nodeVisibility or .linkVisibility I constantly get an error whenever my mouse is moved over the graph element. Prevents dragging actions.

ERROR TypeError: Cannot read property 'visible' of undefined
    at intersectObject (three.module.js:44882)
    at Raycaster.intersectObjects (three.module.js:44959)
    at HTMLCanvasElement.onDocumentMouseMove (index.module.js:100)
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:423)
    at Object.onInvokeTask (core.js:17290)
    at ZoneDelegate.push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask (zone.js:422)
    at Zone.push../node_modules/zone.js/dist/zone.js.Zone.runTask (zone.js:195)
    at ZoneTask.push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask [as invoke] (zone.js:498)
    at invokeTask (zone.js:1744)
    at HTMLCanvasElement.globalZoneAwareCallback (zone.js:1770)

To replicate, hide any number of nodes or links on a graph.
example - ```.nodeVisibility((node: any) => node.id === 'root' ? false : true)```

 - OS: Windows 10
 - Browser: Chrome and Firefox

**Comment by veryspry** [comment]
Just a quick note for what it's worth. It seems that for me the `nodeVisibility` method is causing the error, not `linkVisibility`.

So writing something this would error:
```js
ForceGraph3D()(/* DOMNodePointer */).nodeVisibility(node => node.id !== root);
```
But this would be fine:
```js
ForceGraph3D()(/* DOMNodePointer */).linkVisibility(link => link.source !== 'rootNode')'
```

**Comment by vasturiano** [comment]
Thanks for the report @Alexithemia. This is now fixed in `v1.55.10`.
@veryspry indeed only`nodeVisibilty` was the issue. `linkVisibility` was fine.
